src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -144,8 +144,6 @@ private void testEditSessionAction() throws Exception {
                 editedSession.getCourseId(), editedSession.getFeedbackSessionName());
         editedSession.setInstructions(new Text("<p>" + editedSession.getInstructionsString() + "</p>"));
         assertEquals(editedSession.toString(), savedSession.toString());
-        assertEquals("overflow-auto alert alert-success statusMessage",
```

Comment:
Not entirely useless, but not value-adding anything imo. Why would anyone test the _HTML class_ of something?